flow/metrics.yaml

```diff
@@ -30,6 +37,12 @@ components:
             Latency metrics.
           $ref: '#/components/schemas/Flow.Latency.Metrics'
           x-field-uid: 4
+        rx_per_port:
+          description: |-
+            Enables Rx port level disaggregation with metrics tag name set as "rx_port_name".
```

Comment:
... metrics tag name automatically set by implementation as "rx_port_name"  in the flow metric_response ( use exact attribute names from metric response )

Reply:
Also , do we need updates in otg gnmi model to specify that response can be filtered for specific port by specifing the filter for rx_port_name:<port name>  in this exact format . Otherwise, how will gnmi user know what filter to use or expect which has to be parsed to set expectation in tests when checking stats using gnmi ( or otg metrics)

Reply:
For gNMI query path: “/flows/flow[name=f1]/”
expected gNMI response (for the metric response example in PR description):

```
"updates": [
  {
	"Path": "flows/flow[name=f1]",
	"values": {
	  "flows/flow": {
		"open-traffic-generator-flow:name": "f1",
		"open-traffic-generator-flow:state": {                     // Contains the aggregated per-flow stats
		  "counters": {
			"in-pkts": "1000",
			"in-octets": "8000",
			"out-pkts": "1000",
			"out-octets": "8000"
		  },
		  "in-frame-rate": "QKAAAA==",
		  "name": "f1",
		  "out-frame-rate": "QCAAAA==",
		  "transmit": true
		},
		"open-traffic-generator-flow:tag-metrics": {              // Contains the disaggregated per-flow stats
		  "tag-metric": [
			{
			  "name-value": "rx_port_name=p2",
			  "state": {
				"counters": {
				  "in-octets": "4000",
				  "in-pkts": "500"
				},
				"name-value": "rx_port_name=p2",
				"tags": [
				  {
					"tag-name": "rx_port_name",
					"tag-value": {
					  "value-as-string": "p2",
					  "value-type": "STRING"
					}
				  }
				]
			  }
			},
			{
			  "name-value": "rx_port_name=p3",
			  "state": {
				"counters": {
				  "in-octets": "4000",
				  "in-pkts": "500"
				},
				"name-value": "rx_port_name=p3",
				"tags": [
				  {
					"tag-name": "rx_port_name",
					"tag-value": {
					  "value-as-string": "p3",
					  "value-type": "STRING"
					}
				  }
				]
			  }
			}
		  ]
		}
	  }
	}
  }
]
```

Reply:
Description modified to reflect Rx port name location in tagged response.